Index: src/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java
===================================================================
diff --git a/src/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java b/src/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java
deleted file mode 100644
--- a/src/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java	(revision bf271178d0fc067f95cf761a055920e9229ad8d7)
+++ /dev/null	(revision bf271178d0fc067f95cf761a055920e9229ad8d7)
@@ -1,22 +0,0 @@
-package br.com.projeto_escolar.MVC.Model;
-
-import java.util.HashSet;
-import java.util.Set;
-
-public class CadastroDisciplina {
-    private Set<Disciplina> disciplinas = new HashSet<>();
-
-    public void adicionarDisciplina(Disciplina d){
-        disciplinas.add(d);
-    }
-
-    public boolean verificarDisciplina(String codigo){
-        for (Disciplina d : disciplinas){
-            if (d.getCodigo().equals(codigo)){
-             return true;
-            }
-        }
-        return false;
-    }
-
-}
Index: .idea/Projeto_escolar.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"JAVA_MODULE\" version=\"4\">\r\n  <component name=\"CheckStyle-IDEA-Module\" serialisationVersion=\"2\">\r\n    <option name=\"activeLocationsIds\" />\r\n  </component>\r\n  <component name=\"NewModuleRootManager\" inherit-compiler-output=\"true\">\r\n    <exclude-output />\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <sourceFolder url=\"file://$MODULE_DIR$/src/src\" isTestSource=\"false\" />\r\n    </content>\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/Projeto_escolar.iml b/.idea/Projeto_escolar.iml
--- a/.idea/Projeto_escolar.iml	(revision bf271178d0fc067f95cf761a055920e9229ad8d7)
+++ b/.idea/Projeto_escolar.iml	(date 1760974511905)
@@ -6,7 +6,7 @@
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src/src" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
Index: src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java b/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java
new file mode 100644
--- /dev/null	(date 1760974511882)
+++ b/src/br/com/projeto_escolar/MVC/Model/CadastroDisciplina.java	(date 1760974511882)
@@ -0,0 +1,36 @@
+package br.com.projeto_escolar.MVC.Model;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+public class CadastroDisciplina {
+    private Set<Disciplina> disciplinas = new HashSet<>();
+
+    public void adicionarDisciplina(Disciplina d){
+        disciplinas.add(d);
+    }
+
+    public boolean verificarDisciplina(String codigo){
+        for (Disciplina d : disciplinas){
+            if (d.getCodigo().equals(codigo)){
+             return true;
+            }
+        }
+        return false;
+    }
+
+    public void removerDisciplina(String codigo){
+        Iterator<Disciplina> disciplinaIterator = disciplinas.iterator();
+        while (disciplinaIterator.hasNext()){
+            Disciplina disciplina = disciplinaIterator.next();
+            if (disciplina.getCodigo().equals(codigo)){
+                disciplinaIterator.remove();
+                System.out.println(disciplina.getNomeDisciplina());
+                System.out.println(disciplina.getCodigo());
+                System.out.println("removendo...");
+            }
+        }
+    }
+
+}
Index: src/src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java b/src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java
rename from src/src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java
rename to src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java
--- a/src/src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java	(revision bf271178d0fc067f95cf761a055920e9229ad8d7)
+++ b/src/br/com/projeto_escolar/MVC/Model/LeitorDataBase.java	(date 1760974481630)
@@ -1,10 +1,6 @@
 package br.com.projeto_escolar.MVC.Model;
 import java.io.BufferedReader;
 import java.io.FileReader;
-import java.io.IOException;
-
-import br.com.projeto_escolar.MVC.Model.Estudante;
-import br.com.projeto_escolar.MVC.Model.*;
 
 public class LeitorDataBase {
     
diff --git a/src/src/br/com/projeto_escolar/MVC/View/NovaJframe.java b/src/br/com/projeto_escolar/MVC/View/NovaJframe.java
rename from src/src/br/com/projeto_escolar/MVC/View/NovaJframe.java
rename to src/br/com/projeto_escolar/MVC/View/NovaJframe.java
diff --git a/src/src/br/com/projeto_escolar/MVC/Model/Estudante.java b/src/br/com/projeto_escolar/MVC/Model/Estudante.java
rename from src/src/br/com/projeto_escolar/MVC/Model/Estudante.java
rename to src/br/com/projeto_escolar/MVC/Model/Estudante.java
diff --git a/src/src/br/com/projeto_escolar/MVC/Model/Matricula.java b/src/br/com/projeto_escolar/MVC/Model/Matricula.java
rename from src/src/br/com/projeto_escolar/MVC/Model/Matricula.java
rename to src/br/com/projeto_escolar/MVC/Model/Matricula.java
diff --git a/src/src/br/com/projeto_escolar/MVC/Model/ListaEstudantes.java b/src/br/com/projeto_escolar/MVC/Model/ListaEstudantes.java
rename from src/src/br/com/projeto_escolar/MVC/Model/ListaEstudantes.java
rename to src/br/com/projeto_escolar/MVC/Model/ListaEstudantes.java
diff --git a/src/src/br/com/projeto_escolar/MVC/Model/Disciplina.java b/src/br/com/projeto_escolar/MVC/Model/Disciplina.java
rename from src/src/br/com/projeto_escolar/MVC/Model/Disciplina.java
rename to src/br/com/projeto_escolar/MVC/Model/Disciplina.java
